src/modelfree/shaping_wrappers.py

```diff
@@ -90,7 +106,7 @@ class NoisyAgentWrapper(DummyModel):
     def __init__(self, agent, noise_annealer, noise_type='gaussian'):
         """
         Wrap an agent and add noise to its actions
-        :param agent: (BaseRLModel) the agent to wrap
+        :param agent: (DummyModel) the agent to wrap
```

Comment:
Do we depend on it being a `DummyModel`? If not, I think best to keep it `BaseRLModel` in the doc.